nf_core/schema.py

```diff
@@ -167,6 +168,28 @@ def validate_params(self):
         log.info("[green]\[✓] Input parameters look valid")
         return True
 
+    def validate_default_params(self):
+        """
+        Check that all default parameters in the schema are valid
+        Ignores 'required' flag, as required parameters might have no defaults
+        """
+        self.get_schema_defaults()
+        try:
+            assert self.schema is not None
+            # Make copy of schema and remove required flags
+            schema_no_required = copy.deepcopy(self.schema)
+            if "required" in schema_no_required:
+                schema_no_required.pop("required")
+            for group_key, group in schema_no_required["definitions"].items():
+                if "required" in group:
+                    schema_no_required["definitions"][group_key].pop("required")
+            jsonschema.validate(self.schema_defaults, schema_no_required)
+        except AssertionError:
+            log.error("[red]\[✗] Pipeline schema not found")
```

Comment:
I'm pretty sure that these escaping backslashes are no longer needed with Rich. I tried to remove them all the other day but I might have missed some.. (https://github.com/nf-core/tools/pull/831 + https://github.com/nf-core/tools/pull/831/commits/bcf63b14d6c03641155507ed2e90f303129f7ca0 )

```suggestion
            log.error("[red][✗] Pipeline schema not found")
```